O3exp/pipeline/plotting.ini

```diff
@@ -418,11 +418,14 @@ sngl-stat = ${fit_by_template|sngl-stat}
 fit-function = ${fit_by_template|fit-function}
 ; don't try and fit everything down to zero stat value
 stat-fit-threshold = ${fit_by_template|stat-threshold}
+prune-number = 1 
+; note this pruning not the same style as in fit_by_template
```

Comment:
this won't tell people what it is, and assumes they know what fit_by_template does 
suggest 'remove loudest 1 event in each histogram plotted'
(NB comments usually come above the option)